imports/ui/components/error.css

```diff
@@ -1,12 +1,75 @@
+/*@import url('https://fonts.googleapis.com/css?family=Montserrat:300,400,700');*/
+
+.body {
+  color: #333;
+}
+
+.landing1 {
+  background-color: #FFF;
+  font-family: "Montserrat", sans-serif;
+  padding: 50px 0;
+}
+
+.landing1 .row {
+  margin: 0;
+}
+
+.landing1-top {
+  padding: 45px 35px;
+}
+
+.landing1-box {
+  width: 100%;
+  max-width: 750px;
+  background-color: #FFF;
+  box-shadow: 0px 5px 40px rgba(0,0,0,0.07);
```

Comment:
That's a very low and precise number, just wondering what's the rationale / difference if we just use 0.1. Also do you think we should have a single standard number for shadow opacity?

Reply:
This is from the previous landingpage.css.

https://github.com/BRL-CAD/OGV-meteor/blob/development/imports/ui/pages/landingPage.css

Reply:
I think having different shadow opacity does look good in certain places compared to others